src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -43,7 +43,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       ...(appData[ManagingOfficerKey] ?? [])
     ];
 
-    const hasNewlyAddedBosMos = allBosMos.find(boMo => boMo.ch_reference === undefined) !== undefined;
+    const hasNewlyAddedBosMos = allBosMos.find(boMo => !boMo.ch_reference) !== undefined;
```

Comment:
Normally I'd prefer to be explicit about undefined and "" but check ob line 47 is already just using truthy quality. Something is setting the ch_references to "" which is not great but probably outside the scope of this bug fix.